public/user-doc.md

```diff
@@ -0,0 +1,183 @@
+
+![draw rectangle](./images/map-drawing-rectangle-icon.png) &nbsp; To draw a roughly rectangular polygon, click the **Draw a rectangle** tool icon.
+  Click and drag the mouse pointer from one corner of your desired rectangular region to the opposite corner.
+
+![draw polygon](./images/map-drawing-polygon-icon.png) &nbsp; To draw a more elaborate polygon, click the **Draw a polygon** tool icon.
+  Click on the map at each successive corner of the polygon.
+  Finish the polygon by clicking on the first corner again.
+
+![edit shapes](./images/map-drawing-edit-icon.png) &nbsp; To edit a polygon already drawn on the map, click the **Edit shapes** icon.
+  Click and drag the corners of a polygon on the map to adjust its shape. A rectangle's location and dimensions can be changed but not the rectangular shape. A general polygon can be adjusted arbitrarily, including adding new corners by clicking "ghost" corners in the middle of existing edges.
+
+![remove shapes](./images/map-drawing-delete-icon.png) &nbsp; To remove one or more polygons from the map, click the **Remove shapes** icon. 
+  To remove all shapes, click **Remove all**. To remove a single shape, click on it. Repeat as desired. When you have removed all shapes you wish to remove, click **Save**. To cancel the effect of removing one or more individual shapes, click **Cancel**. 
+
+When you add, remove or change a polygon on the map, the station count on the **Station Filters** tab is adjusted to reflect the stations selected. However, _all_ stations selected by the station filters are still shown on the map so that you can adjust your polygon(s) to include or exclude particular stations.
+
+### Filtering stations (**Station Filters** tab)
+
+When the app starts up (i.e., the web page is refreshed), all available stations are loaded and shown on the map. To select stations by criteria including date range of available observations, the network they are part of, the variables they report, and the frequency of observations reported by the station, use the controls on **Station Filters** tab.
+
+![station filters tab](./images/station-filters-tab.png)
+
+At the top of the **Station Filters** tab is a legend that shows the number of stations selected by these controls (and by the map polygons, see above) out of the total available. 
+
+The following controls control the filtering of stations.
+
+**Start Date**, **End Date**: Select only stations with observations within this period. Either date may be unspecified, in which case it means the earliest or latest date available, respectively. Default is empty (no filtering by date).
+
+**Only include stations with climatology**: Some stations have climatology variables associated with them. (These variables are typically computed after the fact from instantaneous values reported by the station). To select only stations for which there are climatology variables, check the checkbox. Default is unchecked.
```

Comment:
Should we add "30 year averages" to clarify what a climatology is for the lay user?